examples/hybrid/bubble.jl

```diff
@@ -104,19 +104,47 @@ function init_dry_rising_bubble_2d(x, z)
 end
 
 # initial conditions
-coords = Fields.coordinate_field(hv_center_space);
-face_coords = Fields.coordinate_field(hv_face_space);
+coords = Fields.coordinate_field(hv_center_space)
+face_coords = Fields.coordinate_field(hv_face_space)
 
 Yc = map(coords) do coord
     bubble = init_dry_rising_bubble_2d(coord.x, coord.z)
     bubble
-end;
+end
 
 ρw = map(face_coords) do coord
     Geometry.Cartesian3Vector(0.0)
-end;
+end
+
+Y = Fields.FieldVector(Yc = Yc, ρw = ρw)
+
+function energy(Yc, ρu, z)
+    ρ = Yc.ρ
+    ρθ = Yc.ρθ
+    u = ρu / ρ
+    kinetic = ρ * norm(u)^2 / 2
+    potential = z * grav * ρ
+    internal = C_v * pressure(ρθ) / (ρ * R_d)
```

Comment:
```suggestion
    internal = (C_v * pressure(ρθ) / (ρ * R_d)) * ρ
```
Sorry when you asked about internal energy I didn't realized you need to multiply it by ρ too.

Reply:
Yes.